setup.py

```diff
@@ -2,7 +2,7 @@
 
 setuptools.setup(
     name='mffpy',
-    version='0.4.0',
+    version='0.4.1',
```

Comment:
We should update this line as well:

https://github.com/BEL-Public/mffpy/blob/3cfc79d05279625d78c56a2b276cb80fcb1f3eac/mffpy/__init__.py#L19

Reply:
Updated!